docfx/Dockerfile

```diff
@@ -12,12 +12,15 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from ubuntu:20.04
+FROM ubuntu:20.04
 
 ENV DEBIAN_FRONTEND noninteractive
 ENV PATH ${PATH}:/opt/docfx
 
-RUN apt-get update && apt-get -y install gnupg ca-certificates wget zip python3
+RUN apt-get update && \
+    apt-get -y install gnupg ca-certificates wget zip python3 python3-pip
+
+RUN python3 -m pip install gcp-docuploader
```

Comment:
It might be tricky to update gcp-docuploader to the newest release because this layer will be cached and reused from the previous build.

Reply:
Options came to mind.

* Install it in `generate.sh`
* Pin the version, then change the version when we want a new release
* Introduce cache busting in the Dockerfile (although I don't know how)

Reply:
Installed in `generate.sh` via `requirements.txt`. I also needed to update `PATH` with the installation location (done in `generate.sh`).